### PR TITLE
Survival charts don't show p-value whenever any group has less than 10 cases

### PR DESCRIPTION
**Pull request by dippindots**
Fix cBioPortal/cbioportal#8719

### CODE REVIEW

File: src/pages/resultsView/survival/SurvivalChart.tsx [hunk 1]
```diff
@@ -341,24 +341,33 @@ export default class SurvivalChart
         return data;
     }
 
+    private get showPValueText() {
+        // 1. p value is not null or undefined
+        // 2. every group has at least 10 cases
+        return (
+            !_.isNil(this.props.pValue) &&
```
Comment (dippindots) [inline src/pages/resultsView/survival/SurvivalChart.tsx:348]: https://lodash.com/docs/4.17.15#isNil

_.isNil(value): Checks if value is null or undefined.

**Review by inodb — APPROVED** [review]
Great, thank you!!

**Review by jjgao — CHANGES_REQUESTED** [review]
We should add a message about why p-value is not shown.

**Comment by inodb** [comment]
@dippindots @jjgao maybe we can show the text:

"p-value not shown if <10 samples"

**Comment by dippindots** [comment]
Thanks @jjgao @inodb for review this! Any suggestions on how to show this message? On our example link, I already see 3 messages on the top...

**Comment by inodb** [comment]
@dippindots thanks so much! Just realized the samples typo on my end, my bad. Maybe instead of

```
Logrank Test P-Value: N/A (<10 samples)
```

We can do 

```
Logrank Test P-Value: N/A (<10 cases)
```

File: src/pages/resultsView/survival/SurvivalChart.tsx [hunk 2]
```diff
@@ -341,26 +341,36 @@ export default class SurvivalChart
...
+    }
+
+    private get pValue() {
+        // show NA if any group has least than 10 cases
+        const showNA = _.some(
+            this.props.analysisGroups,
+            group => this.props.sortedGroupedSurvivals[group.value].length < 10
+        );
+        if (showNA) {
+            return 'N/A (<10 samples)';
```
Comment (inodb) [inline src/pages/resultsView/survival/SurvivalChart.tsx:356]:
```suggestion
            return 'N/A (<10 cases)';
```

**Comment by inodb** [comment]
Maybe we should write this out a bit more - 

Be careful when interpreting this p-value because group x has less than y samples

**Comment by dippindots** [comment]
close this until we have better understanding of how to fix this.

**Pull request closed without merging**